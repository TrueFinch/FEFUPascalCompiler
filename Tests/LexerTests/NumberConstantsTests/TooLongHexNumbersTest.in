$AB89F1234DCEF128391273AB182C