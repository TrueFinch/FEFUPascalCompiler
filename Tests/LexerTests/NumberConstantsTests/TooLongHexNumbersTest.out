Error on (1,1) in $AB89F1234DCEF128391273AB182C: Value was either too large or too small for a UInt32.